tests/TestCase/View/Helper/TextHelperTest.php

```diff
@@ -634,4 +634,36 @@ public function testAutoParagraph()
         $result = $this->Text->autoParagraph(null);
         $this->assertTextEquals($expected, $result);
     }
+
+    /**
+     * testSlug method
+     *
+     * @param string $string String
+     * @param array $options Options
+     * @param String $expected Expected string
+     * @return void
+     * @dataProvider slugInputProvider
+     */
+    public function testSlug($string, $options, $expected)
+    {
+        $result = $this->Text->slug($string, $options);
+        $this->assertSame($expected, $result);
+    }
+
+    /**
+     * @return array
+     */
+    public function slugInputProvider(): array
+    {
+        return [
+            [
+                'Foo Bar: Not just for breakfast any-more', [],
```

Comment:
The split array lines here looks weird, but don't know that it matters for tests.

Reply:
Copy and paste from utility tests